app/service/base_service.py

```diff
@@ -79,3 +80,8 @@ class Reason(Enum):
         FACT_DEPENDENCY = 2
         OP_RUNNING = 3
         UNTRUSTED = 4
+
+    @staticmethod
+    async def load_module(module_type, module_info):
```

Comment:
can you change the operation_svc to use this as well? you can do this in a separate PR.

Reply:
Yes I'll do that